### PR TITLE
feat: update map for mobile use

### PR DESCRIPTION
**Pull request by pyphilia**
This PR adds a new route `/map` that will display the map "in full screen". This is used by the mobile app to display the map.

### CODE REVIEW

File: src/components/item/MapView.tsx [hunk 1]
```diff
@@ -9,18 +9,21 @@ import { buildPlayerTabName } from '@/config/selectors';
 
 type Props = {
   parentId?: DiscriminatedItem['id'];
-  title: string;
+  title?: string;
+  height?: string;
 };
 
-const MapView = ({ parentId, title }: Props): JSX.Element => {
+const MapView = ({ parentId, title, height = '80vh' }: Props): JSX.Element => {
```
Comment (spaenleh) [inline src/components/item/MapView.tsx:16]: The container is a Flex item, so it should be possible to grow it to fill the page.

File: src/config/paths.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ export const ITEM_ID_PARAMS = 'itemId';
 export const ITEM_SHARE_PATH = 'share';
 export const ITEM_PUBLISH_PATH = 'publish';
 export const ITEM_SETTINGS_PATH = 'settings';
+export const MAP_ITEMS_PATH = 'map';
```
Comment (spaenleh) [inline src/config/paths.ts:15]: So it will be `https://builder.graasp.org/map?parentId=1234` ?

File: src/components/App.tsx [hunk 3]
```diff
@@ -81,9 +83,11 @@ const App = (): JSX.Element => {
     RecycledItemsScreen,
     withAuthorizationProps,
   );
+  const MapWithAuthorization = MapItemsScreen;
```
Comment (spaenleh) [inline src/components/App.tsx:86]: So no auth currently ?

**Review by spaenleh — APPROVED** [review]
Looks good, just some questions

File: src/components/App.tsx (hunk 3 above)
Comment (pyphilia) [reply src/components/App.tsx:86]: you can access the map without auth, if the item is public.

**Review by ReidyT — APPROVED** [review]
Looks good 👍

**Pull request merged**